UserReportManager.kt

```diff
@@ -1,3 +1,13 @@
 object UserReportManager {
 
+    val userReportService: UserReportService = UserReportService()
+
+    fun sendAllUsersReports(users: List<User>) {
+        for (user in users) {
+            for (userReport in user.reports.orEmpty()) {
+                userReportService.sendReport(userReport)
```

Comment:
I see a discrepancy with the implementation and acceptance criteria. Did something change?

Reply:
Good catch! our PM changed the criteria in our Slack chat yesterday, I'll ping her to update the criteria.